src/pybind/mgr/dashboard/controllers/rbd_mirroring.py

```diff
@@ -296,26 +298,36 @@ def _get_content_data():  # pylint: disable=R0914
         for mirror_image in mirror_images:
             image = {
                 'pool_name': pool_name,
-                'name': mirror_image['name']
+                'name': mirror_image['name'],
+                'state_color': mirror_image['state_color'],
+                'state': mirror_image['state']
             }
 
             if mirror_image['health'] == 'ok':
                 image.update({
-                    'state_color': mirror_image['state_color'],
-                    'state': mirror_image['state'],
                     'description': mirror_image['description']
                 })
                 image_ready.append(image)
             elif mirror_image['health'] == 'syncing':
-                p = re.compile("bootstrapping, IMAGE_COPY/COPY_OBJECT (.*)%")
-                image.update({
-                    'progress': (p.findall(mirror_image['description']) or [0])[0]
-                })
+                if mirror_image['state'] == 'Replaying':
+                    p = re.compile("replaying, ({.*})")
+                    replaying_data = p.findall(mirror_image['description'])
+                    assert len(replaying_data) == 1
+                    replaying_data = json.loads(replaying_data[0])
+                    seconds_until_synced = 0
+                    if 'seconds_until_synced' in replaying_data:
+                        seconds_until_synced = replaying_data['seconds_until_synced']
+                    image.update({
+                        'seconds_until_synced': seconds_until_synced
+                    })
```

Comment:
There's more info that could be useful to the front-end, right?

What about defining this via NamedTuple?
```py
class ReplayingData(NamedTuple):
  bytes_per_second: Optional[int] = None
  seconds_until_synced: Optional[int] = None
  syncing_percent: Optional[float] = None
```
And then:
```suggestion
                    for field in ReplayingData._fields:
                        try:
                            image[field] = replaying_data[field]
                        except KeyError:
                            pass
```
Or (the following might look redundant, but it's basically ensuring that the untyped `replaying_data` dict is validated via `ReplayingData`; this could be improved with TypedDicts, but that's not available in 3.6):
```suggestion
                    image.update(**ReplayingData(**replaying_data)._asdict())
```